server/routes/api/contacts/index.js

```diff
@@ -96,7 +96,7 @@ router.get('/:id', isAuthenticated, (req, res) => {
     });
 });
 
-router.put('/edit/:id', isAuthenticated, (req, res) => {
+router.put('/:id/edit', isAuthenticated, (req, res) => {
```

Comment:
You don't need the word edit at all here. The fact that you're making a put is enough.